scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/FindRuns.java

```diff
@@ -0,0 +1,86 @@
+package ca.on.oicr.gsi.runscanner.scanner;
+
+import ca.on.oicr.gsi.runscanner.dto.type.Platform;
+import ca.on.oicr.gsi.runscanner.scanner.processor.RunProcessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.TimeZone;
+import java.util.stream.Collectors;
+
+/**
+ * Attempts to process run directories, provided on the command line, through getRunsFromRoot() and
+ * display the results. This is for debugging purposes.
+ */
+public final class FindRuns {
+
+  public static void main(String[] args) throws IOException {
+    if (args.length == 0) {
+      System.err.println(
+          "Usage: java -DplatformType=ILLUMINA -Dname=default -Dtz=America/Toronto -Dparameters={} ca.on.oicr.gsi.runscanner.scanner.Test_getRunsFromRoot /path/to/run/folder");
```

Comment:
I think the class name in this usage message is wrong Please also document this utility in the readme